src/Hypergrid.js

```diff
@@ -1151,7 +1151,7 @@ var Hypergrid = Base.extend('Hypergrid', {
      * @returns {boolean} The given row is fully visible.
      */
     isDataRowVisible: function(r) {
-        return this.renderer.isRowVisible(r);
+        return this.renderer.getVisibleDataRow(r);
```

Comment:
@hugoeanogueira Are you suggesting here that isRowVisible needs to be subgrid aware?